ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/event_handlers.schema.yml

```
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  event_handlers:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    display_name: Event Handlers
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          display_name: Event Handler Name
+        action_type:
+          type: str
+          display_name: Action Type
+          description: |
+            Type of action
+          valid_values: ["bash", "increment", "log"]
+        action:
+          type: str
+          description: |
+            Command to execute
+        delay:
+          type: int
+          description: |
+            Event-handler delay in seconds
+        trigger:
+          type: str
+          description: |
+            Configure event trigger condition. Only supports on-logging
+        regex:
+          type: str
+          description: |
+            Regular expression to use for searching log messages. Required for on-logging trigger
+        asynchronous:
+          type: bool
+          default: False
+          description: |
+            Set the action to be non-blocking. if unset, default is False
```

The default column captures this information, IMO we don't need to repeat it here!
```suggestion
            Set the action to be non-blocking.
```